CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2023-09-03
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* No new clusters can be created with [Azure AD Integration (legacy)](https://learn.microsoft.com/azure/aks/azure-ad-integration-cli). Existing AKS clusters with Azure Active Directory integration will keep working. All Azure AD Integration (legacy) AKS clusters will be migrated to [AKS-managed Azure AD](https://learn.microsoft.com/azure/aks/managed-azure-ad) automatically starting from 1st Dec. 2023. We recommend updating your cluster with AKS-managed Azure AD before 1 Dec 2023. This way you can manage the API server downtime during non-business hours.
+* OSsku=cblmariner (Mariner 1.0) reached end of life on July 31, and is no longer receiving security updates, in accordance with Mariner's 2 year lifecycle for major versions. Please migrate to OSsku=azurelinux (Mariner 2.0). Read more on [Mariner 2.0][https://microsoft.github.io/CBL-Mariner/announcing-mariner-2.0/]
```

Comment:
Might be removed, waiting on azlinux team

Reply:
Can we fix the link [ ] should be ( )